### ISSUE TITLE
Spark Integration Issues

### ISSUE THREAD

**Issue by clgy15** [aishfenton/Vegas#42]
I've been trying to use case class RDDs in Jupyter, but I can't seem to load or call the right function

```Scala
import vegas._
import vegas.render.HTMLRenderer._
implicit val displayer: String => Unit = { s => kernel.display.content("text/html", s) }
import vegas.spark.Spark._

Vegas("NAME OF CHART").withRDD(RDD[TESTCLASS]).<More parameters>.show
```
Name: Compile Error
Message: <console>:97: error: value withRDD is not a member of vegas.DSL.SpecBuilder
              Vegas("helloworld").withRDD(subsample2).show
                                  ^
StackTrace: 

You're readme needs to be updated to. Your import statement is wrong, missing the spark part. Also the the spark section is wrong where you say you can call withData and pass an RDD in. The code in the spark module shows its withRDD. So which is it? I can't seem to get this to work, I would love to get this to work. Any help would be appreciated.

**Comment by dbtsai** [comment]
Interesting. It's using implicit to convert the RDD or DataFrame locally for vega.

**Comment by aishfenton** [comment]
@clgy15, Ok, I think we figured it out now. Sorry for the confusion. We just released 0.3.3 that should address this.
